joint_mappings:
  - axis: 0
    min_velocity: -1.0
    max_velocity: 1.0
    invert: false

  - axis: 1
    min_velocity: -1.0
    max_velocity: 1.0
    invert: false

  - axis: 2
    min_velocity: -1.0
    max_velocity: 1.0
    invert: false

  - axis: 5
    min_velocity: -1.0
    max_velocity: 1.0
    invert: false
